nilearn/plotting/find_cuts.py

```diff
@@ -106,10 +108,16 @@ def find_xyz_cut_coords(img, mask_img=None, activation_threshold=None):
         mask = mask[slice_x, slice_y, slice_z]
         my_map *= mask
         offset += [slice_x.start, slice_y.start, slice_z.start]
-
     # Testing min and max is faster than np.all(my_map == 0)
     if (my_map.max() == 0) and (my_map.min() == 0):
-        return .5 * np.array(data.shape)
+        warnings.warn(
+            "Could not determine cut coords: "
+            "All values were masked. "
+            "Returning center of mass of unmasked data instead.")
+        cut_coords = ndimage.center_of_mass(np.abs(data)) + offset
```

Comment:
Not sure, but when you do that, the affine is not applied: only the translation "offset" part is applied, not the linear transformation. It worked if the affine is eye(4) but not in general ?